Lib/test/test_array.py

```diff
@@ -1174,6 +1249,8 @@ def test_iadd(self):
     def test_imul(self):
         super().test_imul()
```

Comment:
Can the ones around here that no longer have the `@unittest.expectedFailure` decorators on them be removed? @frank-king left these behind without realizing why I put them there.

Reply:
@fanninpm you don't need to mention someone unless we actually need their attention. In this case, I think referring commit or pr url can be helpful, but not about the author.